fhir-bulkdata-webapp/pom.xml

```diff
@@ -45,6 +45,23 @@
             <groupId>org.glassfish</groupId>
             <artifactId>jakarta.json</artifactId>
         </dependency>
+        <dependency>
+            <!-- This is provided, yet never used. -->
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-server</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
wondering what this is for... are we just using it for its transitive dependency on other things marked provided?

Reply:
since we pick up operation-bulkdata, I don't want to pull in fhir-server.  I marked as provided, and it is not used.